Makefile

```diff
@@ -279,12 +279,25 @@ clean-downloadcache:
 update-downloadcache: $(CACHE)
 	cd $(CACHE) && git pull origin master || true
 
+.PHONY: collect-requirements
+collect-requirements:
+	-@rm -rf $(COLLECTED_REQUIREMENTS)
+	@mkdir $(COLLECTED_REQUIREMENTS)
+	bin/pip install -d $(COLLECTED_REQUIREMENTS) --no-compile --no-index --no-dependencies --find-links $(WHEEL_CACHE) --find-links $(PYTHON_CACHE) -r requirements.txt
+	# Arch-specific wheels cannot be used.
+	-@rm -rf $(COLLECTED_REQUIREMENTS)/zope.interface*
+	-@rm -rf $(COLLECTED_REQUIREMENTS)/MarkupSafe*
+	-@cp $(CACHE)/python/zope.interface* $(COLLECTED_REQUIREMENTS)
```

Comment:
It is exceedingly weird and I should've documented it.

pip does grab them but without being asked turns them into architecture-specific wheels.  I see know way to make it stop.

So the ugly alternative was to manually note the ones that are trouble, delete the wheel, and replace with a tarball.